provider/main.py

```diff
@@ -74,7 +78,7 @@ def on_create(event):
         data_type = fileType
         if not data_type:
             data_type = s3Path.rsplit('.', 1)[-1]
-        secrets = sops_decode(raw_content, kmsKey, data_type)
+        secrets = sops_decode(raw_content, data_type, kmsKey)
```

Comment:
Oh god I've just realised what a horrible job of the camel-vs-snake casing I've done in this file who even am I?

Reply:
ikr